cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -28,7 +25,7 @@ const (
 	DefaultKubernetesFallbackVersion = "v1.5.0"
 	DefaultAPIBindPort               = 6443
 	DefaultDiscoveryBindPort         = 9898
-	DefaultAuthorizationMode         = authorizer.ModeRBAC
+	DefaultAuthorizationMode         = "RBAC"
```

Comment:
@liggitt Please omit this line when you upload your new changeset, I'm changing this in https://github.com/kubernetes/kubernetes/pull/40149